tket1-passes/README.md

```diff
@@ -7,26 +7,19 @@ API.
 
 ## Building instructions
 
-Building `tket1-passes` is done using `conan >= 2.0`. After installing `conan`,
-(`pip install conan` or use `pipx`), you must set up `conan` to use the
-artifactory containing the tket package:
-
-```
-conan remote add tket-libs https://quantinuumsw.jfrog.io/artifactory/api/conan/tket1-libs --index 0
```

Comment:
Are you not using the jgrof remote anymore?

Reply:
Yes I am, but I'm (sneakily) adding it in the `tket1-passes/build.rs` build script (see line 28), so that nothing is required from the user.